nucypher/cli/actions.py

```diff
@@ -306,8 +312,10 @@ def make_cli_character(character_config,
 
 def select_stake(stakeholder, emitter) -> Stake:
     stakes = stakeholder.all_stakes
-    enumerated_stakes = dict(enumerate(stakes))
-    painting.paint_stakes(stakes=stakes, emitter=emitter)
+    active_stakes = sorted((stake for stake in stakes if stake.is_active),
```

Comment:
:+1: 
Much needed to sort them!

Reply:
Btw I am wondering: is it possible to have stakes sorted in the order addresses are derived by geth?

Reply:
I guess we could try to preserve that order, but I don't know if geth enforces some ordering criteria. I think that alphabetical order is fine here, tbh.

Reply:
Would also agree that alphabetical is fine here.